colcon_core/shell/template/prefix_util.py.em

```diff
@@ -17,6 +17,12 @@ FORMAT_STR_USE_ENV_VAR = '@(shell_extension.FORMAT_STR_USE_ENV_VAR)'
 @{assert shell_extension.FORMAT_STR_INVOKE_SCRIPT is not None}@
 FORMAT_STR_INVOKE_SCRIPT = '@(shell_extension.FORMAT_STR_INVOKE_SCRIPT)'
 
+@[if hasattr(shell_extension, 'FORMAT_STR_REMOVE_TRAILING_SEPARATOR')]@
```

Comment:
As soon as the constant is added / documented to the extension point (see other comment) this check needs to compare against `None` instead.

Reply:
I guess we don't need to check since it will be defined as `None` in the extension point.

Reply:
Updated 9cba5a5